unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ProfileHUD/ProfileHUDController.cs

```diff
@@ -88,6 +91,13 @@ public struct Configuration
             view.ManaPurchasePressed += (object sender, EventArgs args) => WebInterface.OpenURL(URL_MANA_PURCHASE);
         }
 
+        myAccountCardController = new MyAccountCardController(
```

Comment:
why are we instantiating the controller here? cant we inject it?

Reply:
+1

Reply:
I did a proper injection of the controller & view here: https://github.com/decentraland/unity-renderer/pull/5335/commits/50d5e2e14200095eb746c9a2dc3ea4907a16cd39
However doing the test coverage of `ProfileHUDController`, since its legacy code, requires a refactor of the majority of the class in the first place. It has many static calls, and should proper handle the dependency injection of each collaborator. I think it gets out of the scope of this PR

Reply:
I have just refactorized all the static calls related to `WebInterface` injecting the `IBrowserBridge` interface. We would have to continue refactorizing some more static calls in the class, but I agree with Nico that it's out of the scope of this PR.

Reply:
yeah, that HUD is the worst. It's fine!